tests/utils/kudo.go

```diff
@@ -229,7 +231,7 @@ func (c *KubernetesTestClient) PrintLogsOfNamespace(namespace string) {
 	log.Info(fmt.Sprintf("Using kubectl from path: %s", kubectlPath))
 
 	cmd := exec.Command(kubectlPath, "logs", fmt.Sprintf("-l heritage=kudo"), fmt.Sprintf("--namespace=%s", namespace))
-	log.Infoln(fmt.Sprintf("logs %s --heritage=kudo", namespace))
+	log.Infoln(fmt.Sprintf("logs %s --heritage=kudo -c %s", namespace, suites.DefaultContainerName))
```

Comment:
It seems you're only changing what's logged here, without changing the actual command on preceding line?

Reply:
you are right! let me create the string command so nobody get confused with this again